ddtrace/contrib/requests/patch.py

```diff
@@ -13,7 +13,7 @@
 
 # requests default settings
 config._add('requests', {
-    'service_name': get_env('requests', 'service_name', default=DEFAULT_SERVICE),
+    "service_name": config.get_service(default=get_env("requests", "service_name", default=DEFAULT_SERVICE)),
```

Comment:
Is this _just_ for server-side? Or will this apply to all clients that use `requests`?

Reply:
This will apply to `requests` in general. Let me know if this is not what we want 😛

Reply:
cc @brettlangdon I'm pretty sure we don't want the requests client's service name to be overridden (not yet anyway).

Also, I've never used `requests` as a server-side framework before. I didn't know you could?